README.md

```diff
@@ -1,2 +1,77 @@
 # PowerUp [![Build Status](https://travis-ci.org/MontclairRobotics/PowerUp.svg?branch=master)](https://travis-ci.org/MontclairRobotics/PowerUp)
-2018 Competition Season
+2018 FRC PowerUp Code
+
+## Table of Contents
+1. [IntelliJ set up](#intellij-set-up)
+2. [Eclipse set up](#eclipse-set-up)
+3. [GradleRIO command reference](#gradlerio-command-reference)
+
+## IntelliJ set up
```

Comment:
Make a note telling the user to close out of IntelliJ if they have it open because this can cause problems. As well as and instruction on what to do if things don't work